app/src/main/java/com/onboarding/wheatherapp/services/response/DataResponse.kt

```diff
@@ -0,0 +1,6 @@
+package com.onboarding.wheatherapp.services.response
+
+data class DataResponse(
+    val dt: String,
```

Comment:
whats dt stands for?

Reply:
Is the time of data forecasted, It's not necessary, I will remove it